### ISSUE TITLE
Support for lambdifying logical operators with multiple arguments

### ISSUE THREAD

**Issue by adeak** [sympy/sympy#12128]
While logical operators such as `Or` and `And` work fine for more than 2 input arguments, the resulting function from `lambdify` tends to choke on this. Example, motivated by [this question](http://stackoverflow.com/q/42045906/5067311) on Stack Overflow:

```
from sympy import symbols, Or, lambdify
x,y,z = symbols('x y z')
expr = Or(x, y, z)
fun = lambdify((x,y,z), expr)
fun(False, False, True)
```

The above throws a confusing error (at least confusing in case numpy is installed), since the third input argument is misunderstood by numpy.

Since `Or` and `And` with more than two input arguments could only be mapped to nested binary operations if we stick to `or/and`-type maps, I instead suggest doing something to the effect

```
fun = lambdify((x,y,z), expr, modules={'Or':any, 'And':all})
```

by default (unless there's a practical reason not to). Admittedly operators such as `Xor` would be harder to generalize.

**Comment by oscarbenjamin** [comment]
This seems to be fixed in master:
```julia
In [65]: from sympy import symbols, Or, lambdify 
    ...: x,y,z = symbols('x y z') 
    ...: expr = Or(x, y, z) 
    ...: fun = lambdify((x,y,z), expr) 
    ...: fun(False, False, True)                                                                                                                              
Out[65]: True
```

**Comment by adeak** [comment]
@oscarbenjamin indeed, thank you, fixed already on the 1.3 release I have installed. Closing it.
